dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporterTest.java

```diff
@@ -136,9 +141,519 @@ public void testAddPrefix() throws IOException {
+  @Test
+  void testLongSumLinesDelta() throws IOException {
+    ByteArrayOutputStream bos = new ByteArrayOutputStream();
+    ByteArrayInputStream bis =
+        new ByteArrayInputStream(
+            "{\n\"linesOk\": 3,\n\"linesInvalid\": 0,\n  \"error\": null\n}".getBytes());
+
+    HttpURLConnection connection = setUpMockConnection(202, bos, bis);
+
+    Collection<LongPointData> longPointDataCollection =
+        new ArrayList<LongPointData>() {
+          {
+            add(
+                LongPointData.create(
+                    1619687639000000000L, 1619687659000000000L, Attributes.empty(), 123L));
+            add(
+                LongPointData.create(
+                    1619687639000000000L, 1619687659000000000L, Attributes.empty(), 321L));
+            add(LongPointData.create(0L, 0L, Attributes.empty(), 456L));
+          }
+        };
+    LongSumData longSumData =
+        LongSumData.create(true, AggregationTemporality.DELTA, longPointDataCollection);
+
+    MetricData metricData =
+        MetricData.createLongSum(
+            Resource.getDefault(),
+            InstrumentationLibraryInfo.empty(),
+            "longSumData",
+            "",
+            "",
+            longSumData);
+
+    DynatraceMetricExporter metricExporter =
+        DynatraceMetricExporter.builder()
+            .setApiToken("mytoken")
+            .setUrl(connection.getURL())
+            .build();
+
+    CompletableResultCode result =
+        metricExporter.export(Collections.singleton(metricData), connection);
+
+    verify(connection).setRequestMethod("POST");
+    verify(connection).setRequestProperty("Authorization", "Api-Token mytoken");
+    verify(connection).setRequestProperty("Content-Type", "text/plain; charset=utf-8");
+    assertEquals(
+        "longSumData,dt.metrics.source=opentelemetry count,delta=123 1619687659000\n"
+            + "longSumData,dt.metrics.source=opentelemetry count,delta=321 1619687659000\n"
+            + "longSumData,dt.metrics.source=opentelemetry count,delta=456",
+        bos.toString());
+    assertEquals(CompletableResultCode.ofSuccess(), result);
+  }
+
+  @Test
+  void testDoubleSumLinesCumulative() throws IOException {
+    ByteArrayOutputStream bos = new ByteArrayOutputStream();
+    ByteArrayInputStream bis =
+        new ByteArrayInputStream(
+            "{\n\"linesOk\": 2,\n\"linesInvalid\": 0,\n  \"error\": null\n}".getBytes());
+
+    HttpURLConnection connection = setUpMockConnection(202, bos, bis);
+
+    Collection<DoublePointData> doublePointDataCollection =
+        new ArrayList<DoublePointData>() {
+          {
+            add(
+                DoublePointData.create(
+                    1619687639000000000L, 1619687659000000000L, Attributes.empty(), 100.3));
+            add(
+                DoublePointData.create(
+                    1619687639000000000L, 1619687659000000000L, Attributes.empty(), 300.6));
+            add(DoublePointData.create(0L, 0L, Attributes.empty(), 500.8));
```

Comment:
Is there a reason why you didn't put timestamps on these?

Reply:
Not really, I basically copy the lines from the `createDoubleSumLinesCumulative` in the Serializer tests.

Reply:
It leads to different lines, one with and one without timestamp, right? So i guess we can leave it like this, as this will then also test serialization with and without timestamp.

Reply:
Exactly. I'll add a comment so it's clear it was intentional.